roles/firewall_config/handlers/main.yml

```diff
@@ -1,2 +1,5 @@
 ---
-# handlers file for firewall_config
\ No newline at end of file
+# handlers file for firewall_config
+- name: Reload firewall
+  shell: firewall-cmd --reload
```

Comment:
Is there a particular reason you need `shell`? Can you use `command` instead? [Ansible docs](https://docs.ansible.com/ansible/latest/modules/shell_module.html#notes) recommend shell only when absolutely necessary.

Reply:
No particular reason. It is more of a habit, I have to change that.